test/e2e/cross_os/test_job_submissions.py

```diff
@@ -32,6 +32,335 @@
 @pytest.mark.usefixtures("session_worker")
 @pytest.mark.parametrize("operating_system", [os.environ["OPERATING_SYSTEM"]], indirect=True)
 class TestJobSubmission:
+
+    @pytest.mark.parametrize(
+        "append_string_script",
+        [
+            (
+                "#!/usr/bin/env bash\n\n  echo -n $(cat {{Param.DataDir}}/files/test_input_file){{Param.StringToAppend}} > {{Param.DataDir}}/output_file\n"
```

Comment:
Why the different `"` vs `'''` Is it because of the `"` in the windows branch?

Reply:
Yes, it's mostly because of that. By the way this test (and the one after) was just moved to the beginning of this file for this PR. It's been already merged a few weeks ago.